docs/build-docs.sh

```diff
@@ -17,11 +17,22 @@ mkdir docs/build/
 cp -r docs/_templates docs/conf.py docs/*.svg docs/*.json  docs/build/
 
 if [ "$action" == "linkcheck" ]; then
-  sphinx-build -c docs/ -WETan -j auto -D language=en -b linkcheck docs/build/ docs/build/html
+  sphinx-build -c docs/ -ETan -j auto -D language=en -b linkcheck docs/build/ docs/build/html
 elif [ "$action" == "docs" ]; then
-  sphinx-build -c docs/ -WETa -j auto -D language=en docs/build/ docs/build/html
+  sphinx-build -c docs/ -ETa -j auto -D language=en docs/build/ docs/build/html
 fi
 
 # Clean up build artefacts
 rm -rf docs/build/html/_sources
-rm -rf docs/build/[0-9][0-9]_*
+
+# Copy built HTML to temp directory, clean up build dir and replace with built docs only
+rm -rf docs/temp
+mkdir docs/temp/
+mkdir docs/temp/html
+cp -rf docs/build/html/* docs/temp/html
+
+rm -rf docs/build
+mkdir docs/build
+mkdir docs/build/html
+cp -rf docs/temp/html/* docs/build/html
+rm -rf docs/temp
```

Comment:
I'm cleaning out all the other build artifacts...because the build makes a copy of all the markdown sources and also a copy of the markdown sources as .txt files. When you're working with the docs regularly and grepping through, you hit all those as well as the _actual_ sources, and it's really easy to update on by mistake and then wonder why your changes never got in the build (because they're `.gitignore`d). Ask me how I know this 🤣 

There's probably a more elegant way to clean out the build file and just retain the HTML, but that was my quick approach -- copy what you need, delete the whole lot, then move it back.

WDYT? @noklam ?